lxd/device/unix_hotplug.go

```diff
@@ -149,31 +149,42 @@ func (d *unixHotplug) Start() (*deviceConfig.RunConfig, error) {
 	runConf := deviceConfig.RunConfig{}
 	runConf.PostHooks = []func() error{d.Register}
 
-	device := d.loadUnixDevice()
-	if d.isRequired() && device == nil {
-		return nil, fmt.Errorf("Required Unix Hotplug device not found")
+	devices := d.loadUnixDevices()
+	if d.isRequired() && len(devices) <= 0 {
+		return nil, fmt.Errorf("Required unix hotplug device %q not found", d.name)
```

Comment:
Ah wait, GH was munging the two changes together, this is in Start().

Still, I don't think we should be including the device name in the error.

Because we can see from the call site, returns like this:

```
return nil, fmt.Errorf("Failed to start device %q: %w", dev.Name(), err)
```